### PR TITLE
Menu driven calculator

### PR DESCRIPTION
**Pull request by prashantagno3**
This Pull Request resolves Issue #555

Description:
In this Pull Request we have done the following additions/edits for the code:

### CODE REVIEW

File: Participants/menu_calc/menu_calculator.c [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  case '-': res=a-b;
+	  break;
+  case '*': res=a*b;
+	  break;
+  case '/': res=a/b;
+	  break;
+  default: 
+	  printf("\ninvalid choice");
+  }
+printf("result=%.2f\n",res);
```
Comment (utkarsh23) [inline Participants/menu_calc/menu_calculator.c]: Good job! Just add a newline character after `invalid choice` and reindent your code. Also its better to have two separate print statements for taking in the user input for the two numbers.

**Comment by utkarsh23** [comment]
Any update, @prashantagno3 ?

**Comment by salman-bhai** [comment]
Are there any updates here or should we close the PR and label it as `spam`?

**Comment by salman-bhai** [comment]
spam!

**Pull request closed without merging**